### DISCUSSION TITLE
Execute fitting process

### DISCUSSION THREAD

**Discussion by zrazi** [qMRLab/qMRLab#482] [General]
How can I plot the representative fit curve and execute the fitting process? I understand there are ways to do that by using qMRshowOutput(FitResults,data,Model); based on qMRlab instruction, but how would you do it the GUI way?

**Comment by mathieuboudreau** [comment]
Hi @zrazi,

See this demo from last year's ISMRM: https://www.youtube.com/watch?v=V3LMOTuwMs4
